START TIME: Wed May 21 22:22:50 CEST 2025
slurmstepd: error: couldn't chdir to `/workspace': No such file or directory: going to /tmp instead
slurmstepd: error: couldn't chdir to `/workspace': No such file or directory: going to /tmp instead
slurmstepd: error: couldn't chdir to `/workspace': No such file or directory: going to /tmp instead
slurmstepd: error: couldn't chdir to `/workspace': No such file or directory: going to /tmp instead
slurmstepd: error: couldn't chdir to `/workspace': No such file or directory: going to /tmp instead
2025-05-21 22:23:11,311 - root - INFO - Total RAM: 854.46 GB
2025-05-21 22:23:11,311 - root - INFO - Total RAM: 854.46 GB
2025-05-21 22:23:11,311 - root - INFO - Total RAM: 854.46 GB
2025-05-21 22:23:11,311 - root - INFO - Total RAM: 854.46 GB
2025-05-21 22:23:11,311 - root - INFO - Available RAM: 772.53 GB
2025-05-21 22:23:11,311 - root - INFO - Available RAM: 772.53 GB
2025-05-21 22:23:11,311 - root - INFO - Available RAM: 772.53 GB
2025-05-21 22:23:11,311 - root - INFO - Available RAM: 772.53 GB
2025-05-21 22:23:11,920 - root - INFO - Loading a model with scale=19, scaling_strategy=ScalingStrategy.ALL, config:
TransformerModelArgs(dim=4864, n_layers=152, n_heads=152, n_kv_heads=8, multiple_of=256, ffn_dim_multiplier=1.3, norm_eps=1e-05, rope_theta=500000, norm_type='rmsnorm', seq_len=4096, vocab_size=131072)
2025-05-21 22:23:11,920 - root - INFO - Loading a model with scale=19, scaling_strategy=ScalingStrategy.ALL, config:
TransformerModelArgs(dim=4864, n_layers=152, n_heads=152, n_kv_heads=8, multiple_of=256, ffn_dim_multiplier=1.3, norm_eps=1e-05, rope_theta=500000, norm_type='rmsnorm', seq_len=4096, vocab_size=131072)
2025-05-21 22:23:11,921 - root - INFO - Loading a model with scale=19, scaling_strategy=ScalingStrategy.ALL, config:
TransformerModelArgs(dim=4864, n_layers=152, n_heads=152, n_kv_heads=8, multiple_of=256, ffn_dim_multiplier=1.3, norm_eps=1e-05, rope_theta=500000, norm_type='rmsnorm', seq_len=4096, vocab_size=131072)
2025-05-21 22:23:11,924 - root - INFO - Loading a model with scale=19, scaling_strategy=ScalingStrategy.ALL, config:
TransformerModelArgs(dim=4864, n_layers=152, n_heads=152, n_kv_heads=8, multiple_of=256, ffn_dim_multiplier=1.3, norm_eps=1e-05, rope_theta=500000, norm_type='rmsnorm', seq_len=4096, vocab_size=131072)
2025-05-21 22:26:02,553 - root - INFO - Total model parameters: 46,322,328,320
2025-05-21 22:26:02,553 - root - INFO - Took 2 min 51 sec
2025-05-21 22:26:02,553 - root - INFO - 


2025-05-21 22:26:02,692 - root - INFO - Total model parameters: 46,322,328,320
2025-05-21 22:26:02,692 - root - INFO - Took 2 min 51 sec
2025-05-21 22:26:02,692 - root - INFO - 


2025-05-21 22:26:03,348 - root - INFO - Total model parameters: 46,322,328,320
2025-05-21 22:26:03,348 - root - INFO - Took 2 min 52 sec
2025-05-21 22:26:03,348 - root - INFO - 


2025-05-21 22:26:05,236 - root - INFO - Total model parameters: 46,322,328,320
2025-05-21 22:26:05,236 - root - INFO - Took 2 min 53 sec
2025-05-21 22:26:05,236 - root - INFO - 


Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/dist-packages/torch/distributed/run.py", line 922, in <module>
    main()
  File "/usr/local/lib/python3.12/dist-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 355, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/torch/distributed/run.py", line 918, in main
    run(args)
  File "/usr/local/lib/python3.12/dist-packages/torch/distributed/run.py", line 909, in run
    elastic_launch(
  File "/usr/local/lib/python3.12/dist-packages/torch/distributed/launcher/api.py", line 138, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/torch/distributed/launcher/api.py", line 264, in launch_agent
    if result.is_failed():
       ^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'is_failed'
srun: error: nid006791: task 1: Exited with exit code 1
srun: Terminating StepId=456097.0
slurmstepd: error: *** STEP 456097.0 ON nid006791 CANCELLED AT 2025-05-21T22:26:18 ***
srun: error: nid006791: tasks 2-3: Terminated
srun: Force Terminated StepId=456097.0
